main.go

```diff
@@ -31,9 +31,9 @@ import (
 	"github.com/linki/chaoskube/util"
 )
 
-var (
-	version = "undefined"
-)
+const envVarPrefix = "CHAOS_"
```

Comment:
Let's use `CHAOSKUBE_` to make it more specific and not conflict with other tools in an undefined way.